collect_app/src/main/java/org/odk/collect/android/preferences/FormMetadataFragment.java

```diff
@@ -80,8 +82,16 @@ private void initPrefFromProp(PropertyManager propertyManager,
         return new Preference.OnPreferenceChangeListener() {
             @Override
             public boolean onPreferenceChange(Preference preference, Object newValue) {
-                EditTextPreference changedTextPref = (EditTextPreference) preference;
                 String newValueString = newValue.toString();
+
+                if (KEY_METADATA_EMAIL.equals(key)) {
+                    if (!isEmailAddressValid(newValueString)) {
+                        ToastUtils.showLongToast(getString(R.string.invalid_email_address));
```

Comment:
`getString(R.string.invalid_email_address)` this can be replaced with `R.string.invalid_email_address`